_pages/security/dynamic-scanning.md

```diff
@@ -5,63 +5,38 @@ parent: Security
 
 [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) is the tool we use for doing automated vulnerability scanning.
```

Comment:
Sounds like we use OWASP ZAP for both "automated vulnerability scanning" and "manual vulnerability scanning" (according to the rest of the page). Let's change this 

Let's start this page with a super plain language summary, something like this:

> In order for an application to get ATO, it needs to meet more than a minimum level of application security, so the application team needs to run [both static and dynamic security scans](https://pages.18f.gov/before-you-ship/security/scanning/) and document good results. Running a "dynamic" scan means running a program that analyzes a live running application for common vulnerabilities.

And then include this revised OWASP ZAP sentence:

> At 18F, we use [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) as our tool for dynamic vulnerability scanning. If you (or another person on your application team) has questions about setting this up, ask #compliance for help.